guides/doc-Managing_Hosts/topics/proc_upgrading-rhel7-hosts-to-rhel8.adoc

```diff
@@ -0,0 +1,24 @@
+[id="upgrading-hosts-from-rhel7-to-rhel8_{context}"]
+= Upgrading Hosts from RHEL 7 to RHEL 8
+
+You can use a job template to upgrade your Red{nbsp}Hat Enterprise{nbsp}Linux 7 hosts to Red{nbsp}Hat Enterprise{nbsp}Linux 8.
+
+.Prerequisites
+
+* Ensure that your RHEL 7 hosts meet the requirements for the upgrade to RHEL 8. For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/planning-an-upgrade_upgrading-from-rhel-7-to-rhel-8[Planning an upgrade] in the _Upgrading from RHEL 7 to RHEL 8_ guide.
+* Prepare your hosts for the upgrade. For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/preparing-a-rhel-7-system-for-the-upgrade_upgrading-from-rhel-7-to-rhel-8[Preparing a RHEL 7 system for the upgrade] in the _Upgrading from RHEL 7 to RHEL 8_ guide.
+* Enable remote execution feature on {Project}. For more information, see xref:Running_Jobs_on_Hosts[].
+* Distribute {Project} SSH keys to the hosts that you want to upgrade. For more information, see xref:sect-Managing_Hosts-Establishing_a_Secure_Connection_for_Remote_Commands[].
```

Comment:
Another step should be installing Leapp on the host (if it is not covered in the DOCs)

Reply:
The step that does `yum install leapp leapp-repository` is present in the [Prepare your hosts for the upgrade](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/preparing-a-rhel-7-system-for-the-upgrade_upgrading-from-rhel-7-to-rhel-8), which is linked in on line 9.